### ISSUE TITLE
laser broke

### ISSUE THREAD

**Issue by sqlartist** [lilscan/lilscan-grpc#7]
hi, my laser cable has broken off, can you tell me the correct part I need to buy in the UK.

I think this is correct?

https://www.amazon.co.uk/ARCELI-650nm-Module-Driver-Plastic/dp/B07QDS8XZT

**Comment by lilscan** [comment]
Yes, these ones are correct but a bit overpriced. I would consider a price <4€ per laser in Europe (without shipping) as a fair price.
